usr/share/rear/format/USB/default/30_format_usb_disk.sh

```diff
@@ -30,12 +30,15 @@ if [[ "$answer" == "Yes" || "$FORCE" ]]; then
     if [[ "$EFI" == "y" ]]; then
         LogPrint "Creating new vfat filesystem on ${RAW_USB_DEVICE}1"
         mkfs.vfat $v -F 16 -n REAR-EFI ${RAW_USB_DEVICE}1 >&2
+        
+        # create link for EFI partition in /dev/disk/by-label
```

Comment:
unlike mkfs.ext3 (and friends), mkfs.vfat does not seems to update udev so calling partprobe again was needed, hope it is not a big problem ...